metricbeat/docs/modules/aws.asciidoc

```diff
@@ -32,8 +31,14 @@ see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[Te
 aws> sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@example.com --token-code 456789 --duration-seconds 129600
 ----
 
-Specific permissions needs to be added into the IAM user's policy to authorize Metricbeat to collect AWS monitoring metrics. Please
-see documentation under each metricset for required permissions.
+Since temporary security credentials are short term, after they expire, the user needs to generate new ones and modify
```

Comment:
This paragraph sounds like someone could just replace the credentials and it would reload. This is true if reload is enabled, but by default it's off.

Reply:
https://github.com/elastic/beats/pull/11142